apps/events/utils.py

```diff
@@ -71,8 +71,12 @@ def _handle_waitlist_bump_payment(payment, attendees):
                 "Dette arrangementet krever betaling og du har 48 timer på å betale"
             )
 
-    elif payment.payment_type == 3:  # Delay
+    elif payment.payment_type == PaymentTypes.DELAY:  # Delay
         deadline = timezone.now() + payment.delay
+
+        if payment.deadline:
+            deadline = max(deadline, payment.deadline)
+
```

Comment:
This needs to be handled more places than just the waitlist bumping. We should probably change `create_payment_delay` directly, but ensure that it still behaves the same for `PaymentTypes` that are not `utsettelse`

I think the calculated deadline is used later on so the method should probably return the `PaymentDelay` it creates

Reply:
Okay I stand somewhat corrected. If the payment type is `DEADLINE`, and the person is bumped from the waitlist with less than 2 days to the deadline (or when the deadline has passed), they already get at least a two day `payment_delay`. I was thinking about the case where you register _directly_, without getting bumping, which should behave the same (if the payment delay has passed), but you currently just get an expired deadline, and get kicked out once mommy runs.

In other words: we don't need to change anything, just use `DEADLINE`. 

Signing up directly when the payment deadline has passed (`handle_attend_event_payment`), and the payment type is `DEADLINE`, it should probably behave like getting bumped